htdocs/include/defines.php

```diff
@@ -67,7 +67,7 @@
 define('_XOOPS_FATAL_MESSAGE', "<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Strict//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd'>
 <html xmlns='http://www.w3.org/1999/xhtml'>
 <head>
-<meta http-equiv='Content-Type' content='text/html; charset=utf-8;charset=utf-8' />
+<meta http-equiv='Content-Type' content='text/html; charset=utf-8;' />
```

Comment:
Definitely a good change :smile:

That define comes up a lot. It was recently mentioned on XOOPS/XoopsCore25#357 

If we are changing it, we should move it to HTML5, as that is supposed to be out standards. If we can do something easily to make it so it can safely be customized, that would be even better.